cfg/admin.txt

```diff
@@ -122,4 +122,4 @@ excessiveuseofcobby role=admin
 plizzard role=admin
 octareenroon91 role=admin
 serpentarium role=admin
-
+cebutris role=admin
```

Comment:
unrelated but how do I stop this from happening all the time

Reply:
you can ignore it, this file does jack shit on live servers

Reply:
I know, I'd just rather not have it show up on all my pull requests. It's like an annoying ghost that just kinda stands in the corner being annoying, and I can't get it to stop.